addons/bestja_frontend_fpbz/templates/pages/regulations.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This should be put in the `bestja_page_fixtures_fpbz` module. (If it does not exist, it should be created).